quickstart/prerequisites.md

```diff
@@ -8,29 +8,272 @@ All worker nodes must satisfy the following requirements:
 * Linux kernel **5.13 or higher** with the following modules loaded:
   * nvme-tcp
   * ext4 and optionally xfs
+* * [Helm](https://helm.sh/docs/intro/install/) version must be v3.7 or later.
 
-## **Mayastor pods (Data Plane/Engine)**
-Each worker node which will host an instance of a Mayastor pod must have the following resources _free and available_ for _exclusive_ use by that pod:
 
-* Two CPU cores
-* 4GiB RAM
-* **HugePage support**
-  * A minimum of **2GiB of** **2MiB-sized** pages
```

Comment:
Is the above not needed?

Reply:
The above has been added under 'recommended resource requirements'. (from line 31)

Reply:
Those seem to be the resource requests and limits per Mayastor component in K8s.
Above is a generic requirement for the node as a whole.

Reply:
Okay. I'll add it back.